q2_quality_filter/test/test_filter.py

```diff
@@ -116,6 +116,82 @@ def test_q_score(self):
         self.assertEqual(sorted(obs), sorted(exp_trunc))
         pdt.assert_frame_equal(stats, exp_trunc_stats.loc[stats.index])
 
+    def test_q_score_real(self):
+        ar = Artifact.load(self.get_data_path('real_data.qza'))
+        view = ar.view(SingleLanePerSampleSingleEndFastqDirFmt)
+        obs_result, stats = q_score(view, min_quality=39,
+                                    min_length_fraction=0.24)
+
+        # all truncated reads shown, manually identified. comments denote why
+        # the sequence is not retained.
+        exp_result = [
+                      # this record is 25% of full length
+                      "@HWI-EAS440_0386:1:32:15467:1432#0/1",
+                      "TACGGAGGATCCGAGCGTTATCCGGATTTATTGGGTTT",
+                      "+",
+                      "hhhhhhhhhhhhfghhhghghghhhchhhahhhhhfhh",
+
+                      # too short
+                      # "@HWI-EAS440_0386:1:36:9986:17043#0/1",
+                      # "TACGTAGGTGGCAAGCGTTATCCGGATTTATTG",
+                      # "+",
+                      # "hhhhhhhhhhhhhhhhhhhhhhhhhffhhghhh",
+
+                      "@HWI-EAS440_0386:1:37:13343:14820#0/1",
+                      ("TACGGAGGATCCGAGCGTTATCCGGATTTATTGGGTTTAAAGGGAGCGTAGAT"
+                       "GGATGTTTAAGTCAGTTGTG"),
+                      "+",
+                      ("hhhhhhhhhhhhhfhhhhhfhhhhghhhhghhhhhhhhhgghhhgghhhgghh"
+                       "hgdhhhhghghhhdhhhhgh"),
+
+                      "@HWI-EAS440_0386:1:41:18215:15404#0/1",
+                      "TACGTAGGTGGCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCATGTA",
+                      "+",
+                      "hhhhhhhhhhhhghhhhhhhhhhhhffhhghhhhghhghgghghhhhhgh",
+
+                      # too short
+                      # "@HWI-EAS440_0386:1:42:5423:19606#0/1",
+                      # "TACGTAGGGAGCAAGCGTT",
+                      # "+",
+                      # "hhhhghhhhhhhhhghhfh",
+
+                      "@HWI-EAS440_0386:1:52:7507:5841#0/1",
+                      "TACGGAGGATCCGAGCGTTATCCGGATTTATTGGGTTT",
+                      "+",
+                      "hhhhhhhhhghhfghhhhhhhhhhgfhhhghhhghdhh",
+
+                      "@HWI-EAS440_0386:1:53:18599:4074#0/1",
+                      "TACGTAGGTGGCAAGCGTTGTCCGGATTTACTGGGTG",
+                      "+",
+                      "hhhhfhhhhhfhhhhhhfhffhghhfgghggghdcbh",
+
+                      # too short
+                      # "@HWI-EAS440_0386:1:55:16425:9514#0/1",
+                      # "TACGGAGGATCCGAGCGTTATCCGGATT",
+                      # "+",
+                      # "hhhhhhhhhhhhfghhhghghhhhhbgh",
+
+                      "@HWI-EAS440_0386:1:65:12049:5619#0/1",
+                      "TACGTAGGTGGCAAGCGTTATCCGGATTTACTGGGTGTAAAGGGCGTG",
+                      "+",
+                      "hhhhhhhhhhhhhhhhhhhhhhhhhfhhhhhhhghdhghhhhhghcfh",
+
+                      # @HWI-EAS440_0386:1:95:4837:16388#0/1
+                      # starts off < Q40
```

Comment:
The sequence starts off with "g" which is 39 in PHRED+64, and quality filter in QIIME 1.9.1 was defined as `x <= min_quality` (see [here](https://github.com/biocore/qiime/blob/master/qiime/split_libraries_fastq.py#L92))